### DISCUSSION TITLE
Bug: command prompt in Win+X menu not working - request: W11 start menu icon in W10 taskbar style

### DISCUSSION THREAD

**Discussion by zoosterix** [valinet/ExplorerPatcher#672] [General]
Hello, this mod is fantastic, thank you! Anyway I'm here to report a bug and also to request an option.
1) as per subject, command promt link in win+x menu is not working, neither enabling powershell option and restarting explorer.
2) look and behavior of W10 taskbar option is just fantastic but, since all the icons and the start menu are shown in W11 design, I think it would be more coherent and more aestethic to have at least an option to set the original W11 icon for start menu with W10 taskbar.
Thanks

**Comment by valinet** [comment]
Hi

1. May indeed be hit or miss. Idk, they broke something about it in Windows 11. You can use a Win-X menu editor to bring back which ever option you'd like.
2. This is a highly requested feature. I have already implemented it, it just hasn't made it to a stable release yet. You can upgrade to the latest pre-release version and you'll see the option there, or just wait a while until the option makes its way to the next stable release.\\

Thanks

**Reply by zoosterix** [reply]
Ok, installed pre release and enabled w11 icon style. Anyway that icon is not accurate. The real W11 start menu icon is bigger, blueish and with rounded corners. Is it possible to fix it?

**Comment by chip33** [comment]
The Start button can be set to Windows 11 style here. 
![Screenshot 2022-01-16 002844](https://user-images.githubusercontent.com/48499740/149642577-d7ea8dba-9d3d-4e1c-abab-e4c5640cc7c8.png)

**Reply by chip33** [reply]
yes, in the pre release
